diff -ru assimp--2.0.863-sdk/code/CMakeLists.txt assimp--2.0.863-sdk.bak/code/CMakeLists.txt
--- assimp--2.0.863-sdk/code/CMakeLists.txt	2010-11-21 22:40:42.000000000 +0100
+++ assimp--2.0.863-sdk.bak/code/CMakeLists.txt	2011-06-09 13:08:14.000000000 +0200
@@ -453,7 +453,7 @@
 	../contrib/unzip/unzip.h
 )
 
-ADD_LIBRARY( assimp SHARED
+ADD_LIBRARY( assimp STATIC
 	3DSConverter.cpp
 	3DSHelper.h
 	3DSLoader.cpp
@@ -663,26 +663,6 @@
 	../contrib/irrXML/irrTypes.h
 	../contrib/irrXML/irrXML.cpp
 	../contrib/irrXML/irrXML.h
-	../contrib/zlib/adler32.c
-	../contrib/zlib/compress.c
-	../contrib/zlib/crc32.c
-	../contrib/zlib/crc32.h
-	../contrib/zlib/deflate.c
-	../contrib/zlib/deflate.h
-	../contrib/zlib/inffast.c
-	../contrib/zlib/inffast.h
-	../contrib/zlib/inffixed.h
-	../contrib/zlib/inflate.c
-	../contrib/zlib/inflate.h
-	../contrib/zlib/inftrees.c
-	../contrib/zlib/inftrees.h
-	../contrib/zlib/trees.c
-	../contrib/zlib/trees.h
-	../contrib/zlib/zconf.h
-	../contrib/zlib/zconf.in.h
-	../contrib/zlib/zlib.h
-	../contrib/zlib/zutil.c
-	../contrib/zlib/zutil.h
 	../contrib/ConvertUTF/ConvertUTF.c
 	../contrib/unzip/crypt.h
 	../contrib/unzip/ioapi.c
